### ISSUE TITLE
Display Lang options in local language

### ISSUE THREAD

**Issue by ivedula** [universalcore/unicore#143]
Linguists have requested if Español can be displayed instead of Spanish. Will be useful to push this change for other local languages too.

**Comment by miltontony** [comment]
This is part of a larger discussion that we decided to ignore up until now.
The question raised in the beginning was whether or not we want to show "English | Spanish" or whether we want to show Country Flags or abbreviations or something else. This needs to be raised again with Jonathan Mckay, Steve and Studio. 

One of the other reasons why this is contentious is because of the need to differentiate between dialects e.g `Español (Colombia)` vs `Español (México)`. We can often get away with just showing the language (because our sites are territory based), but there's nothing preventing a site from specifying `eng_US` and `eng_GB` which would lead to `English | English`
cc: @smn @SteveBarnett

**Comment by SteveBarnett** [comment]
+1 on showing language selection in the language itself (e.g. Español instead of Spanish). No need for flags, I think: flags are for countries, not for languages.
Some linkies: [UX.Stack Exchange](http://ux.stackexchange.com/questions/37017/language-of-language-names-in-the-language-selector), [Accessibility legend, Roger Johansson](http://www.456bereastreet.com/archive/200604/indicating_language_choice_flags_text_both_neither/)

@miltontony: for `en_US` and `en_GB` we could show `English (US)` and `English (UK)`. That differentiation is important (but more so for Spanish, Portuguese, ...)

**Comment by miltontony** [comment]
@SteveBarnett I agree with you. My point was that we chatted about it briefly in the beginning but didn't make a final descion. There was also a question on the UI practicalities - abbreviations vs `English (United States)` - and how that would look on the UI. Especially in countries like India where we'll have more than 5 languages to select from.
![screenshot from 2015-01-14 12 28 24](https://cloud.githubusercontent.com/assets/623417/5737399/e89b9224-9be8-11e4-976d-6e6362605edc.png)

**Comment by SteveBarnett** [comment]
@miltontony Ah, cool. I can help with figuring out how to get it into the UI. Will keep an eye on it for when it enters a sprint. :)
